docs/usage/shoot_status.md

```diff
@@ -95,8 +95,8 @@ The Shoot status also contains information about the last occurred error(s) (if
 
 Known error codes are:
 
-- `ERR_INFRA_UNAUTHORIZED` - indicates that the last error occurred due to invalid infrastructure credentials. It is classified as a non-retryable error code.
-- `ERR_INFRA_INSUFFICIENT_PRIVILEGES` - indicates that the last error occurred due to insufficient infrastructure privileges. It is classified as a non-retryable error code.
+- `ERR_INFRA_UNAUTHENTICATED` - indicates that the last error occurred due to invalid authentication credentials for the requested resource. It is classified as a non-retryable error code.
+- `ERR_INFRA_UNAUTHORIZED` - indicates that the last error occurred due to lack of authorization to perform the requested operation on the given resource. It is classified as a non-retryable error code.
```

Comment:
Looking at it again I kind of like following the definition of the http error codes (here equivalent to [403](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403))
```suggestion
- `ERR_INFRA_UNAUTHORIZED` - indicates that the last error occurred due to the server understanding the request but refusing to authorize it. It is classified as a non-retryable error code.
```